unlock-js/jest.config.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  collectCoverage: true,
```

Comment:
This proves to be useful in assessing coverage. For now I am not setting a hard number for coverage but we shouldeventually do it.

Reply:
Note that coverage for the `v13/createLock.test.js` is 100%!